src/Explore/DocumentItem.js

```diff
@@ -22,81 +31,144 @@ function DocumentItem(props) {
         overflow: 'hidden',
       }}
     >
-      <Card width={1} key={pid}>
-        <Flex
+      <Card width={1} key={pid} pb={isDetailed ? 0 : 1}>
+        <Box
           sx={{
-            mb: 2,
-            justifyContent: 'space-between',
-            flexWrap: 'wrap-reverse',
+            cursor: 'pointer',
+            ':hover [data-title]': { textDecoration: 'underline' },
+          }}
+          onClick={(evt) => {
+            evt.currentTarget.querySelector('a').click()
           }}
         >
-          <Box
+          <Flex
+            sx={{
+              mb: 2,
+              justifyContent: 'space-between',
+              flexWrap: 'wrap-reverse',
+            }}
+          >
+            <Flex sx={{ color: 'primary', alignItems: 'center', fontSize: 1 }}>
```

Comment:
The DOI is no longer a link to avoid having to tab through twice. Since the entire top-half of the card is interactive, this doesn't matter. In other words, clicking/hovering the DOI works as before (except it is not underlined).